shuup/front/build_resources.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * This file is part of Shuup.
+ *
+ * Copyright (c) 2012-2018, Shuup Inc. All rights reserved.
+ *
+ * This source code is licensed under the OSL-3.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const { getParcelBuildCommand, runBuildCommands } = require("shuup-static-build-tools");
+
+runBuildCommands([
+    getParcelBuildCommand({
+        cacheFolder: "front",
```

Comment:
It's a directory, not a folder. :D   (At least for consistency with the other argument names, this should be cacheDir)